frontend/src/Organize/OrganizePreviewModalContent.js

```diff
@@ -93,7 +95,10 @@ class OrganizePreviewModalContent extends Component {
     return (
       <ModalContent onModalClose={onModalClose}>
         <ModalHeader>
-          Organize & Rename {seasonNumber != null && <SeasonNumber seasonNumber={seasonNumber} />}
+          { seasonNumber ?
+            translate('OrganizeModalHeaderSeason', { season: formatSeason(seasonNumber) }) :
+            translate('OrganizeModalHeader')
+          }
```

Comment:
I think this'll need to be:

```suggestion
          { seasonNumber == null ?
            translate('OrganizeModalHeader') :
            translate('OrganizeModalHeaderSeason', { season: formatSeason(seasonNumber) })
          }
```

otherwise specials are also skipped, but I like the approach of splitting them to different translations.